report/3.Theory/ParserImplementation.tex

```diff
@@ -0,0 +1,96 @@
+\section{Parser implementation}
+
+Since the compiler language kotlin is completely interoperable with java, both SableCC and ANLTR could be obvious choices for parser if the group decided to go with a generated parser. 
```

Comment:
I think you should make it clearer with our compiler language, something along the lines of:
"Since the chosen compiler implementation language, kotlin, is completely......"

Also when referreing to SableCC and ANTLR mayhaps refer back to Lexer (as I describe atleast ANTLR) Right now it seems pretty strange.